changelog-versions/8.0.0.md

```diff
@@ -63,3 +63,23 @@
 - Theme variant logic [#579](https://github.com/Telefonica/mistica-design/issues/579)
 - → Hero component | Design documentation [#986](https://github.com/Telefonica/mistica-design/issues/986)
 - → Improve header doc with slot case [#926](https://github.com/Telefonica/mistica-design/issues/926)
+
+
+## Version notes 
+
+### Figma update strategy
+
+#### Header
+
+We suggest that for new needs of the Header component, the new one be used and the old one be kept in those places where it was used until now.
+
+In the event an update is necessary:
+
+* Only pre-title and title content will be kept when swapping between components
+* It is recommended to extract the rest of the elements and group them in a component before doing the swap, later they can be transferred to the slot of the new component
+
+
+#### NavBar
+
+* The instructions of the component indicate which values the two deprecated should have, this will avoid that when they are eliminated the variant is no longer linked to the component.
+* By using the new boolean properties of Figma both the Profile and the Right icon will be hidden by default, a selection can be made of all the instances of the file to show them again, the content will not suffer any modification.
```

Comment:
:robot: OpenAI

The version notes seem to be well-documented and provide clear instructions on how to use the updated components. However, it would be helpful to include some screenshots or diagrams to better illustrate the suggested changes. Additionally, it may be useful to provide more context around why these changes were made and what benefits they bring to the user experience.
```suggestion

<!-- This is an auto-generated comment by OpenAI -->